atomate/vasp/powerups.py

```diff
@@ -287,6 +288,37 @@ def add_modify_incar(
         )
     return original_wf
 
+def add_modify_kpoints(
+    original_wf, modify_kpoints_params=None, fw_name_constraint=None
+):
+    """
+    Every FireWork that runs VASP has a ModifyKpoints task just beforehand. For
+    example, allows you to modify the KPOINTS based on the Worker using env_chk
+    or using hard-coded changes.
+
+    Args:
+        original_wf (Workflow)
+        modify_kpoints_params (dict) - dict of parameters for ModifyKpoints.
```

Comment:
Can you change the hyphens ("-") to colons, as otherwise these doc string will not be rendered properly in the documentation.

I.e.

```
modify_kpoints_params (dict): dict of parameters for ModifyKpoints.
```

Reply:
Done! Thanks for your help!